builder/table.go

```diff
@@ -158,6 +158,10 @@ func (t Table) WriteColumn(buffer *Buffer, column rel.Column) {
 		buffer.WriteString(" NOT NULL")
 	}
 
+	if column.Primary {
```

Comment:
```
[ CONSTRAINT constraint_name ]
{ NOT NULL |
  NULL |
  CHECK ( expression ) |
  DEFAULT default_expr |
  UNIQUE index_parameters |
  PRIMARY KEY index_parameters |
  REFERENCES reftable [ ( refcolumn ) ] [ MATCH FULL | MATCH PARTIAL | MATCH SIMPLE ]
    [ ON DELETE action ] [ ON UPDATE action ] }
[ DEFERRABLE | NOT DEFERRABLE ] [ INITIALLY DEFERRED | INITIALLY IMMEDIATE ]
```

according to above definition, PRIMARY KEY should be after DEFAULT?
https://www.postgresql.org/docs/9.1/sql-createtable.html

Reply:
no, it's with `|` (or) so order does not matter otherwise `UNIQUE` should also had to be moved after `DEFAULT`

Reply:
Ah got it 🤦